src/reactapp/src/pages/Login.js

```diff
@@ -1,67 +1,110 @@
-import React from 'react';
 import {
-    Flex,
-    Box,
-    FormControl,
-    FormLabel,
-    Input,
-    Checkbox,
-    Stack,
-    Link,
-    Button,
-    Heading,
-    Text,
-    useColorModeValue,
-} from '@chakra-ui/react';
+  Flex,
+  Box,
+  FormControl,
+  FormLabel,
+  InputGroup,
+  InputRightElement,
+  Checkbox,
+  Stack,
+  Link,
+  Button,
+  Heading,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import { useState, React } from "react";
+import { ViewIcon, ViewOffIcon } from "@chakra-ui/icons";
+import { Formik, Form } from "formik";
+
+import InputField from "../components/InputField";
+import { postForm, setToken } from "../components/FetchHelpers";
 
 export default function LoginForm() {
-    return (
-        <Flex
-            minH={'100vh'}
-            align={'center'}
-            justify={'center'}
-            bg={useColorModeValue('gray.50', 'gray.800')}>
-            <Stack spacing={8} mx={'auto'} maxW={'lg'} py={12} px={6}>
-                <Stack align={'center'}>
-                    <Heading fontSize={'4xl'}>Sign in to your account</Heading>
-                    <Text fontSize={'lg'} color={'gray.600'}>
-                        for teacher evaluation tracker
-                    </Text>
+  const [showPassword, setShowPassword] = useState(false);
+
+  const loadEvaluationsPage = () => {
+    // TODO redirect to evaluations landing pa
+  };
+
+  const onSubmitLogin = async (values) => {
```

Comment:
This function demonstrates how to send a request to the API server. In all cases other than this one, we would use `post` rather than `postForm` on line 38.